Tests/Helper/Mocks/MockNetworkService2.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  MockNetworkService2.swift
+//  Hyperspace-iOS
+//
+//  Created by Adam Brzozowski on 1/31/18.
+//  Copyright © 2018 Bottle Rocket Studios. All rights reserved.
+//
+
+import Foundation
+@testable import Hyperspace
+
+class MockNetworkService2: NetworkService {
```

Comment:
Kind of a personal preference, but to me numbering the class doesn't feel right. What about a name like `NetworkServiceMockSubclass` (or some variation of that)?

Reply:
Done